pagopa/io-app
# feat: [IOBP-834] Add pull to refresh into transactions list screen

## ⚠️ This PR depends on #6187 ⚠️

## Short description
This PR adds the refresh control with the pull-to-refresh feature inside the biz-events transaction list screen

## List of changes proposed in this pull request
- Added the `refreshing` property and the `onRefresh` handler to the SectionList.
- Added the `isRefreshing` state that changes when the user pull down to refresh the content.

## How to test
- With the dev-server started, open the `Payments` home page;
- Tap on the `Show all` CTA on the top right corner of the list;
- You should be able to pull to refresh the content when inside that list;

## Preview
https://github.com/user-attachments/assets/9ebbecc3-c507-466b-ad87-ec852d16a85e
